tools/devops/automation/scripts/GitHub.Tests.ps1

```diff
@@ -488,4 +488,83 @@ Describe 'Test-JobSuccess' {
     Context 'unknonw value' {
         Test-JobSuccess -Status "Random value" | Should -Be $False
     }
-}
\ No newline at end of file
+}
+
+Describe 'Get-GitHubPRInfo' {
+    Context 'with all env variables present' {
+
+        BeforeAll {
+            $Script:envVariables = @{
+                "GITHUB_TOKEN" = "GITHUB_TOKEN";
```

Comment:
Should the value `GITHUB_TOKEN` (right side of the assignment) include some sort of value resolution such as $(GITHUB_TOKEN) or $GITHUB_TOKEN)? otherwise, it seems that the token value will then get set to the constant `GITHUB_TOKEN` rather than an actual token

Reply:
is a test. can be any value, we just use the value to check if it is added to the auth header. So the tests sets the env var, used by the cmdlet, we later make sure that the Invoce-RestMethod set the auth header with the token. If we add something that looks like a cred, cred scan will complain and we are just testing that it was something :)